UPCommons.js

```diff
@@ -38,30 +38,32 @@
 
 function detectWeb(doc, url) {
 	var type = ZU.xpathText(doc, '//meta[@name="DC.type"]/@content');
-	if (url.indexOf('/handle/')>-1 && type) {
-		if (itemTypes[type]!=null) return itemTypes[type];
+	if (url.includes('/handle/') && type) {
+		if (itemTypes[type] != null) return itemTypes[type]; // eslint-disable-line no-eq-null
```

Comment:
Oh, we can do better here. The variable `itemType` is defined below and I guess we just have to make sure that `type` is a key of it.